core/kernel/dt.c

```diff
@@ -309,3 +310,23 @@ void _fdt_fill_device_info(void *fdt, struct dt_node_info *info, int offs)
 
 	*info = dinfo;
 }
+
+const void *_fdt_get_secure_prop(const void *fdt, int nodeoffset,
+				 const char *name, int *lenp)
+{
+	const fdt32_t *cuint;
+	char sec_prop[32] = { 0 };
```

Comment:
Minor: `sec_name` for consistency with `name` perhaps

Reply:
ok